internal/services/automation/automation_schedule_resource.go

```diff
@@ -81,14 +82,14 @@ func resourceAutomationSchedule() *pluginsdk.Resource {
 			"interval": {
 				Type:         pluginsdk.TypeInt,
 				Optional:     true,
-				Computed:     true, // defaults to 1 if frequency is not OneTime
+				Computed:     !features.FourPointOhBeta(), // defaults to 1 if frequency is not OneTime
 				ValidateFunc: validation.IntBetween(1, 100),
 			},
 
 			"start_time": {
 				Type:             pluginsdk.TypeString,
 				Optional:         true,
-				Computed:         true,
+				Computed:         !features.FourPointOhBeta(),
```

Comment:
I think it's reasonable to keep properties like `start_time` or `expiry_time` as `Computed` as long as it doesn't interfere with trying to update these properties. Are you able to determine whether updating this property is problematic?